2018-07-15-SciPy-KK/README.md

```diff
@@ -0,0 +1,42 @@
+SciPy 2018 (KK, MT)
+===================
+
+Authors
+-------
+Kacper Kowalik (NCSA, UIUC)
+Matthew Turk (NCSA/iSchool, UIUC)
+
+Short description
+-----------------
+Accessing remote data with persistent, user-definable computational environments is a key challenge in digital scholarship.  The Whole Tale is a response to this, building and combining components to provide a virtual research "home" on the web.  In this talk, we'll describe how the Whole Tale project leverages existing Python projects (Jupyter, PyFilesystem, fusepy, WsgiDAV, Girder and many others) to bring remote analysis and data together, providing persistent workspaces and capturing provenance and data lineage to facilitate the scholarly process.
+
+Long description
+----------------
+
+This talk introduces the Whole Tale project, which aims to enable researchers to examine, transform and then seamlessly republish research data that was used in a scientific articles. We strive to strengthen the three layers of scholarly publication: scholarly process, data access, and computational analysis.
+
+A principle of The Whole Tale is running custom, containerized compute environments, along with external data exposed to the user as a unified FUSE filesystem, transparently mediating access to remote and diverse data sources. We believe that our approach to handling data this way can be easily generalized to other similar projects in the SciPy "ecosystem", such as JupyterHub/BinderHub. Furthermore, introducing an abstract layer between data and computations allows for a finegrained provenace tracking making it easy to provide an automatic credit attribution and boost the reproducibility of the scholarly process.
+
+Our presentation will have the following structure:
+
+* Introducing Whole Tale
```

Comment:
Add (WT) at the end here so when it's referenced later in the bulleted list it's immediately obvious you're referring to WholeTale